kroxylicious/src/main/java/io/kroxylicious/proxy/service/FilterFactoryManager.java

```diff
@@ -11,22 +11,18 @@
 import java.util.ServiceLoader;
 import java.util.Set;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import io.kroxylicious.proxy.config.FilterDefinition;
 import io.kroxylicious.proxy.filter.Filter;
 import io.kroxylicious.proxy.filter.FilterCreationContext;
 import io.kroxylicious.proxy.filter.FilterFactory;
 import io.kroxylicious.proxy.filter.InvalidFilterConfigurationException;
 
 public class FilterFactoryManager {
-    private static final Logger logger = LoggerFactory.getLogger(FilterDefinition.class);
 
+    @Deprecated
```

Comment:
nit: we should add `since` and `forRemoval` (thought quite what thats not implicit in `@Deprecated` I don't know